qa/kerberos/build.gradle

```diff
@@ -84,7 +84,7 @@ dependencies {
 
 // Disable the integration tests for Kerberos until we can find a solution to the failures due to + sign 
 // in the file path on CI.
-boolean disableTests = true
+boolean disableTests = false
```

Comment:
can we remove this boolean and comment ?

Reply:
Actually this should stay disabled for the time being since it is still unlikely to pass general CI without some other changes.